azure-application-insights-spring-boot-starter/build.gradle

```diff
@@ -76,6 +76,23 @@ shadowJar {
         exclude(dependency([group: 'org.apache.httpcomponents', name: 'httpasyncclient', version: '4.1.3']))
         exclude(dependency([group: 'com.google.guava', name: 'guava', version: '20.0']))
         exclude(dependency([group: 'com.google.code.gson', name: 'gson', version: '2.8.2']))
+        
+        // exclude gRPC and protobuf dependencies
```

Comment:
I don't honestly like to have this long list of excluded dependencies. It would have been amazing if shadow jar could excluded transitive dependencies automatically :/